garde/src/rules/range.rs

```diff
@@ -77,7 +77,7 @@ macro_rules! impl_for_int {
     };
 }
 
-impl_for_int!(u8, u16, u32, u64, usize, u128, i8, i16, i32, i64, isize, i128);
+impl_for_int!(u8, u16, u32, u64, usize, u128, i8, i16, i32, i64, isize, i128, f32, f64);
```

Comment:
Could you also add a `f32` field to this test here: 
https://github.com/jprochazk/garde/blob/1c2a13f3e31e269ea84322ea8b281222ea0c0aaa/garde/tests/rules/range.rs#L3-L10

Reply:
Hey sorry for the delay, I updated the test and the snapshot